docs/installation.md

```diff
@@ -0,0 +1,125 @@
+## Installation
```

Comment:
They do contain almost the same stuff, but I wanted to explicitly have all the required steps in the README so there would be a central place for those things. Up for debate, though.